lib/core/src/display/navigation/navigator.rs

```diff
@@ -90,10 +104,22 @@ impl Navigator {
             let y = pan.movement.y * scale;
             let z = 0.0;
 
-            properties_clone.mod_spring(|spring| { spring.fixed_point += Vector3::new(x, y, z); });
-        };
-
-        let dom_clone = dom.clone();
+            properties.mod_spring(|spring| { spring.fixed_point += Vector3::new(x, y, z); });
+        });
+
+        let mut dom_clone = dom.clone();
+        dom_clone.add_resize_callback(
+            enclose!((camera,mut properties) move |dimensions:&Vector2<f32>| {
+                camera.set_screen(dimensions.x, dimensions.y);
+                camera.update();
+                let position = camera.transform().position();
+                properties.mod_kinematics(|kinematics| {
+                    kinematics.set_position(position);
+                    kinematics.set_velocity(Vector3::new(0.0, 0.0, 0.0));
+                });
+                properties.mod_spring(|spring| spring.fixed_point = position);
+            })
+        );
```

Comment:
Do I understand correctly that after resize the movement of the navigator is stopped? If so, its not bad, but is this the only solution here?

Reply:
To sum up my previous comments: yes, that's the solution. You can see there is a small amount of code change in this PR, but I actually spend lots of hours testing it. This is not a workaround, because it considers how the current architecture works. So this is the best solution I got without making a major architectural change.

Reply:
I know, its always hard. I'm ok with this solution. Its not the most beautiful to stop the camera movement, but the resize will happen rarely so its ok :)